galleries/javed/style.css

```diff
@@ -0,0 +1,34 @@
+
+.galleri h1{
+  text-align: center;
+}
+.pics{
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 5px;
+  width: 100%;
+}
+.pic{
+  display: grid;
+  grid-template-rows: 4fr 0.2fr;
+}
+div .img img{
```

Comment:
```suggestion
img{
```

Reply:
You don't need this high level of specificity.  img is enough. or perhaps .img img